GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -499,12 +499,16 @@ void TextureCacheVulkan::BuildTexture(TexCacheEntry *const entry) {
 		imageLayout = VK_IMAGE_LAYOUT_GENERAL;
 	}
 
+	if (plan.saveTexture) {
+		actualFmt = VULKAN_8888_FORMAT;
+	}
+
 	char texName[128]{};
 	snprintf(texName, sizeof(texName), "tex_%08x_%s", entry->addr, GeTextureFormatToString((GETextureFormat)entry->format, gstate.getClutPaletteFormat()));
 	image->SetTag(texName);
 
 	bool allocSuccess = image->CreateDirect(cmdInit, plan.createW, plan.createH, plan.depth, plan.levelsToCreate, actualFmt, imageLayout, usage, mapping);
-	if (!allocSuccess && !lowMemoryMode_) {
+	if (!allocSuccess && !lowMemoryMode_ && !replacer_.Enabled()) {
```

Comment:
Low memory actually happens a lot because of replacements on an underpowered device - why disable the check here?

A user may have replacements + scaling enabled, and this at least makes it decimate more aggressively.

-[Unknown]